src/main/kotlin/game/racingcar/domain/Car.kt

```diff
@@ -2,17 +2,20 @@ package game.racingcar.domain
 
 import game.racingcar.strategy.MoveStrategy
 
-class Car {
-    var blackBox = arrayListOf<Int>()
-    var position: Int = 0
+class Car(val name: String) {
+    private val _blackBox: MutableList<Int> = mutableListOf()
+    private var _position: Int = 0
+
+    val blackBox: List<Int>
+        get() = _blackBox.toList()
+    val position: Int
+        get() = _position
 
     fun move(moveStrategy: MoveStrategy): Int {
-        position = moveStrategy.move(position)
-        blackBox.add(position)
-        return position
+        _position = moveStrategy.move(_position)
+        _blackBox.add(_position)
+        return _position
     }
 
-    fun history(time: Int): Int {
-        return blackBox[time]
-    }
+    fun history(time: Int) = blackBox[time]
```

Comment:
```suggestion
    fun history(time: Int): Int = blackBox[time]
```

저는 식으로 된 함수에 대해 리턴 타입이 있는 경우는 항상 붙여주고 있습니다! 왜냐하면 변경에 대해 빠르게 인지하기 위해서입니다. 😆